src/components/dashboard/main-content/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -336,28 +272,79 @@ class BaseCourseCard extends Component {
+    const course = isExecutiveEducation2UCourse ? 'Executive Education' : 'Course';
+    const tooltipText = isExecutiveEducation2UCourse
+      ? 'Executive Education courses are instructor-led, cohort-based, and follow a set schedule.'
+      : 'Courses are on-demand, self-paced, and include asynchronous online discussion.';
+
+    if (orgName) {
+      return (
+        <p className={`mb-2 font-weight-light ${isExecutiveEducation2UCourse ? 'text-light-300' : ''}`}>
+          {orgName} &#x2022; {course}
+          <OverlayTrigger
+            trigger={['hover', 'focus']}
+            placement="top"
+            overlay={(
+              <Tooltip variant="light" id="tooltip-top">
+                {tooltipText}
+              </Tooltip>
+            )}
+          >
+            <InfoOutline className="ml-2 mb-1" style={{ width: '1rem', height: '1rem' }} />
+          </OverlayTrigger>
+        </p>
+      );
+    }
+    return null;
+  };
+
+  renderStartDate = () => {
+    const { startDate } = this.props;
     const formattedStartDate = startDate ? dayjs(startDate).format('MMMM Do, YYYY') : null;
+    const isCourseStarted = dayjs(startDate) <= dayjs();
 
-    if (isExecutiveEducation2UCourse && formattedStartDate) {
-      return <>&#x2022; Start date: {formattedStartDate}</>;
+    if (formattedStartDate && !isCourseStarted) {
+      return <span className="font-weight-light pr-2">Starts {formattedStartDate}</span>;
     }
     return null;
   };
 
-  renderOrganizationName = () => {
-    const { orgName, mode } = this.props;
+  renderEndDate = () => {
+    const { endDate, type } = this.props;
+    const formattedEndDate = endDate ? dayjs(endDate).format('MMMM Do, YYYY') : null;
+    const isCourseStarted = dayjs(this.props.startDate) <= dayjs();
 
-    const isExecutiveEducation2UCourse = EXECUTIVE_EDUCATION_COURSE_MODES.includes(mode);
-    const execEdClass = isExecutiveEducation2UCourse ? 'text-light-300' : '';
-    if (orgName) {
-      return <p className={`mb-0 ${execEdClass}`}>{orgName} {isExecutiveEducation2UCourse && <>&#x2022; Executive Education</>} {this.renderCourseStartDate()}</p>;
+    if (formattedEndDate && isCourseStarted && type !== COURSE_STATUSES.completed) {
+      return <span className="font-weight-light pr-2">Ends {formattedEndDate}</span>;
     }
     return null;
   };
 
+  renderEnrollByDate = () => {
+    const { enrollBy, type } = this.props;
+    const formattedEnrollByDate = enrollBy ? dayjs(enrollBy).format('MMMM Do, YYYY') : null;
+    const isNotEnrolled = type === COURSE_STATUSES.requested || type === COURSE_STATUSES.assigned;
+    // Determines if a user is not enrolled in a course based on the course enrollment status
+    // 'requested' and 'assigned' are not real course statuses.
+
+    if (formattedEnrollByDate && isNotEnrolled) {
+      return <>&#x2022;<span className="font-weight-light pl-2">Enroll by {formattedEnrollByDate}</span></>;
```

Comment:
nit: could use `&bull;` which is a bit more readable for what character it is at first glance